src/main/java/org/candlepin/insights/resource/OpenApiSpecResource.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2009 - 2019 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.insights.resource;
+
+import org.candlepin.insights.exception.ErrorCode;
+import org.candlepin.insights.exception.RhsmConduitException;
+
+import org.springframework.stereotype.Component;
+
+import java.io.InputStream;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+/**
+ * Serves the OpenAPI spec as /openapi.json and /openapi.yaml.
+ *
+ * These are loaded from the API jar.
+ */
+@Component
+@Path("/")
+public class OpenApiSpecResource {
```

Comment:
Might be a good idea to name this something like ApplicationRootResource, or something of the like, just in case we need to add something else down the line.